.github/workflows/static.yml

```diff
@@ -18,10 +18,10 @@ jobs:
       - name: Install
         uses: docker://composer
         with:
-          args: install --ignore-platform-reqs
+          args: install --ignore-platform-reqs --no-ansi --no-suggest
 
       - name: PHPStan
-        uses: docker://oskarstark/phpstan-ga
+        uses: OskarStark/phpstan-ga@0.12.28
```

Comment:
Just some nitpicking, but I think this could stay `docker://oskarstark/phpstan-ga`

Reply:
This is not working :/ check the last build

Reply:
fixed i had a typo